### DISCUSSION TITLE
Get size of resized image?

### DISCUSSION THREAD

**Discussion by CalculationCycle** [imgproxy/imgproxy#809] [Q&A] [answered]
When resizing with %resizing_type set to fit, do I get the exact size of the resulting image somewhere/somehow? I've looked in the headers, hoping to find something there.
Somewhere inside the black box magic of imgproxy the new size is calculated, is it returned somehow?

**Comment by DarthSim** [comment, marked as answer]
I can add this info to the debug headers. Please create a feature request so I can track it.

**Reply by CalculationCycle** [reply]
[Issue](https://github.com/imgproxy/imgproxy/issues/846) created
